daemon/src/maker_cfd.rs

```diff
@@ -439,9 +439,11 @@ where
         // state change. Once we know that we go for either an accept/reject scenario we
         // have to remove the current order.
         self.current_order_id = None;
+
         self.takers
-            .send(maker_inc_connections::BroadcastOrder(None))
+            .do_send_async(maker_inc_connections::BroadcastOrder(None))
```

Comment:
This should need a `allow(clippy::disallowed_method)`?

Reply:
Clippy doesn't complain :O

Reply:
Can you add it anyway? I might be due to some cached artifacts and I don't want to blow up later down the line!

With a comment about the dead lock please.